src/python/base/memoize.py

```diff
@@ -44,7 +44,11 @@ def _redis_client():
     return _local.redis
 
   host = os.getenv('REDIS_HOST', _DEFAULT_REDIS_HOST)
-  port = os.getenv('REDIS_PORT', _DEFAULT_REDIS_PORT)
+
+  # Casting to int as os.getenv returns str
+  # Casting default port to str as default needs to be str or None
+  port = int(os.getenv('REDIS_PORT', str(_DEFAULT_REDIS_PORT)))
```

Comment:
No need of str( on host var. int(int) is also int.

Reply:
without str(). pylint was giving the following error:
`src/python/base/memoize.py:48:13: W1508: os.getenv default type is builtins.int. Expected str or None. (invalid-envvar-default)`
As default val only expects str or None values.

Reply:
from system import environment

environment.get_value('REDIS_PORT', _DEFAULT_REDIS_PORT)

then you don't need to use this bad conversion.

Reply:
Thanks!